autoload/ale/fix/registry.vim

```diff
@@ -32,6 +32,11 @@ let s:default_registry = {
 \       'suggested_filetypes': ['javascript', 'css', 'html'],
 \       'description': 'Apply fecs format to a file.',
 \   },
+\   'buildifier': {
+\       'function': 'ale#fixers#buildifier#Fix',
+\       'suggested_filetypes': ['bzl'],
```

Comment:
This tool can also be used for simple `starlark` scripts, which have a `star` extension.

Reply:
I've haven't seen a Starlark script with a `star` extension. I've only ever seen the `bzl` extension. And at least on Bazel 0.29.1, `bzl` is the only supported file extension for Starlark scripts. Anything else throws an error.

Reply:
I was thinking about `starlark.vim` file type detection in https://github.com/cappyzawa/starlark.vim/blob/master/ftdetect/starlark.vim

I have seen starlark scripts with a `star` extension in a few places, but cannot give any links right now. :)